shiny/ui/_include_helpers.py

```diff
@@ -178,6 +184,73 @@ def include_css(
     return tags.link(dep, href=src, rel="stylesheet")
 
 
+@add_example()
+def include_bootstrap_css(
+    theme: Path | str | HTMLDependency,
```

Comment:
Seems we could keep things a lot simpler if we dropped support for this type. I'm not entirely sure we can get away with that though

```suggestion
    theme: Path | str,
```